### ISSUE TITLE
"rustup self help" should include an inspirational quote

### ISSUE THREAD

**Issue by brson** [rust-lang/rustup#1033]
Suggested by @mbrubeck, who knows I'm a big fan of easter eggs. I approve.

**Comment by brson** [comment]
I should actually think this through harder before deciding we should definitely do it.

**Comment by smackysnacks** [comment]
[ELIZA](https://github.com/AndrewRadev/rust-eliza)!

**Comment by londospark** [comment]
Does anyone have an inspirational quote or a list of quotes that they think should be included?

**Comment by kinnison** [comment]
I'm not sure what would inspire people, though as vaguely useful to programmers go, I quite like:

* “Knowledge is power.” – Francis Bacon
* “ Code is like humour. When you have to explain it, it’s bad.” – Cory House
* “Make it work, make it right, make it fast.” – Kent Beck
* “Any fool can write code that a computer can understand. Good programmers write code that humans can understand.” – Martin Fowler
* “That's the thing about people who think they hate computers. What they really hate is lousy programmers.”
― Larry Niven
* “The most important property of a program is whether it accomplishes the intention of its user.”
― C.A.R. Hoare

I'm sure you could find more with a brief look around.  It'd be worth ensuring that none of the authors of the quotes are now considered problematic before using them though, I'd rather not introduce that kind of problem into the rustup codebase 

(Note, I've not checked the authors above, it's just a set of quotes I quite liked when looking around).

**Comment by kinnison** [comment]
Of course, as was suggested above, thinking carefully about whether or not this would be acceptable to add is also important.  We'd be adding public API to the primary interface a lot of people have with the language, so I'd not immediately accept a PR for this unless it was reasoned out as to why this is safe/sane to add.

**Comment by mbrubeck** [comment]
`rustup self help` is already a valid command, which is the reason I made this joke originally.  This wouldn't add any new API; it would just add a sentence or two to the existing help text.

**Comment by amikai** [comment]
I'd like to take on this task. To clarify the implementation specification, I have a question about where I can place the sentence.
Append the sentence after help text like following image ?
![CleanShot 2023-10-25 at 23 30 09@2x](https://github.com/rust-lang/rustup/assets/13815956/3084096e-76b5-44a7-80fd-5dce2c1e07f0)

**Comment by djc** [comment]
The hard part is figuring out which quote(s) to put in. I'm not sure we currently have the resources to invest in this task.

**Comment by djc** [comment]
> @rustbot claim

I don't think this is a good fit for an external contributor.

@rustbot unassign

**Comment by medzernik** [comment]
:/ okay
